src/app.rb

```diff
@@ -0,0 +1,85 @@
+require './person'
+require './book'
+require './rental'
+require './teacher'
+require './student'
```

Comment:
- [ ] The correct way to invoke a file in the same directory is using `require_relative` instead of `require` it is used to load modules or libraries that are globally installed on the system or are available in Ruby's module search path. Please 🙏🏽 use `require_relative` instead of `require` to call the app class.